components/org.wso2.carbon.consent.mgt.ui/src/main/resources/web/consent/add-purpose.jsp

```diff
@@ -221,22 +221,6 @@
                                         </div>
                                     </td>
                                 </tr>
-                                <tr>
-                                    <td><fmt:message key="purpose.flow"/><%if(!isPurposeGroupPresent) {%><font color="red">*
```

Comment:
Can't completely remove this input. Instead, we have to hide it for ,`if (isPurposeGroupPresent) `

Reply:
fixed